projects/packages/my-jetpack/_inc/state/actions.js

```diff
@@ -21,11 +21,15 @@ const SET_CHAT_AVAILABILITY_IS_FETCHING = 'SET_CHAT_AVAILABILITY_IS_FETCHING';
 const SET_CHAT_AVAILABILITY = 'SET_CHAT_AVAILABILITY';
```

Comment:
This is just a general comment and should not be addressed in this PR. With how many different types of data we are now querying, I think we might want to split up the actions/reducers/selectors/etc into different files so these files don't get so massive they're impossible to parse. Kinda like how it's currently done in the Jetpack plugin. Just a thought 😄